SmarTest/Components/Pages/Login.razor

```diff
@@ -0,0 +1,66 @@
+﻿@page "/"
+@layout Layout.EmptyLayout
+@using SmarTest.Services.Interfaces
+
+<div class="login-page-container">
+    <div class="login-page-inner">
+        <h3>Login</h3>
+        <EditForm Model="@loginModel" OnValidSubmit="HandleLogin">
+            <DataAnnotationsValidator />
+            <ValidationSummary />
+
+            <div class="form-group">
+                <label for="firstName">First Name</label>
+                <InputText id="firstName" class="form-control" @bind-Value="loginModel.FirstName" />
+                <ValidationMessage For="@(() => loginModel.FirstName)" />
+            </div>
+
+            <div class="form-group">
+                <label for="lastName">Last Name</label>
+                <InputText id="lastName" class="form-control" @bind-Value="loginModel.LastName" />
+                <ValidationMessage For="@(() => loginModel.LastName)" />
+            </div>
+
+            <button type="submit" class="btn btn-primary">Login</button>
+        </EditForm>
+
+        <p>@loginMessage</p>
+    </div>
+</div>
+
+@code {
+    private LoginModel loginModel = new LoginModel();
+    private string loginMessage;
+
+    [Inject]
```

Comment:
Check this:
https://learn.microsoft.com/en-us/mobile-blazor-bindings/advanced/dependency-injection